hazelcast-jet-core/src/main/java/com/hazelcast/jet/JetInstance.java

```diff
@@ -134,6 +138,39 @@ default Job getJob(@Nonnull String name) {
         return getJobs(name).stream().findFirst().orElse(null);
     }
 
+    /**
+     * Returns the IMap that contains the data of an exported snapshot with the
+     * given name.
+     * <p>
+     * The contents of the map are unspecified.
+     *
+     * @param name exported snapshot name
+     * @return IMap instance with the data
+     */
+    default IMapJet<Object, Object> getExportedState(@Nonnull String name) {
+        return getMap(Jet.EXPORTED_STATES_PREFIX + name);
+    }
+
+    /**
+     * Returns the collection of names of existing exported states.
+     */
+    default Collection<String> getExportedStateNames() {
```

Comment:
see my comment above. Returning a list of strings without any context would not be useful.